packages/react-composites/tests/browser/common/utils.ts

```diff
@@ -112,7 +112,7 @@ export const waitForPageFontsLoaded = async (page: Page): Promise<void> => {
     //     interface Document {}
     // this breaks `tsc`, even though it works correctly in the browser.
     // eslint-disable-next-line @typescript-eslint/no-explicit-any
-    await (document as any).fonts.ready;
+    return await (document as any).fonts.ready;
```

Comment:
Before this PR, this always returned a fulfilled promise that was always truthy.
So the tests did not actually wait for fonts to load.